docs/legacy/v2.md

```diff
@@ -201,7 +201,7 @@ Auto decoder accept an optional argument `isCamelCase`:
 
 ```fsharp
 > let json = """{ "id" : 0, "name": "maxime", "email": "mail@domain.com", "followers": 0 }"""
-> Decode.Auto.fromString<User>(json, isCamelCase=true)
+> Decode.Auto.fromString<User>(json, caseStrategy=CamelCase)
```

Comment:
This is the legacy documentation so we don't have to update it.

I will also need to "save" the v3 documentation without the new API change.